doc/devices/qvm.rst

```diff
@@ -190,36 +190,4 @@ After measuring the qubit state, we can determine the probability :math:`P_0` of
 
 This process is done automatically behind the scenes in the QVM device when ``qml.expval(qml.Hermitian)`` is returned.
 
-
-
-
-Device options
-~~~~~~~~~~~~~~
-
-On initialization, the PennyLane-Forest devices accept additional keyword
-arguments beyond the PennyLane default device arguments.
-
-``forest_url`` (*str*)
-    the Forest URL server. Can also be set by
-    the environment variable ``FOREST_SERVER_URL``, or in the ``~/.qcs_config``
-    configuration file. Default value is ``"https://forest-server.qcs.rigetti.com"``.
-
-``qvm_url`` (*str*)
-    the QVM server URL. Can also be set by the environment
-    variable ``QVM_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:5000"``.
-
-``compiler_url`` (*str*)
-    the compiler server URL. Can also be set by the environment
-    variable ``COMPILER_URL``, or in the ``~/.forest_config`` configuration file.
-    Default value is ``"http://127.0.0.1:6000"``.
```

Comment:
Same as before, you delete these keyword arguments but then mention them in the next section.